config/config.go

```diff
@@ -10,8 +10,15 @@ import (
 // Config values for the application.
 type Config struct {
 	BindAddr                   string        `envconfig:"BIND_ADDR"`
+	KafkaLegacyAddr            []string      `envconfig:"LEGACY_KAFKA_ADDR"`
+	KafkaLegacyVersion         string        `envconfig:"LEGACY_KAFKA_VERSION"`
```

Comment:
Im not sure where you have used this - does this configuration need to be removed if it isn't used anymore?

Reply:
It also makes me think there is a mistake in the codebase if TLS isn't set will this fail to consume/produce messages?

Reply:
the secrets will need to be updated, initially ("feature-flagged off") with these two as a copy of the existing values

Reply:
I just realised this config must be used in dp-kafka